jobs/tests/templates/bin/pre-start

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+set -eu
+exec 2>&1
+export LANG=en_US.UTF-8
+JOB=etcd-tls
+BIN=pre-start
+JOB_DIR=/var/vcap/jobs/tests/data/$JOB
+CERTS_DIR=/var/vcap/data/targets
+TMP_DIR=/var/vcap/data/tmp/$JOB
+
+# put cfssl in our path
+export PATH=$PATH:/var/vcap/packages/cfssl/bin
+
+echo "[$(date)] $BIN/$$: regenerating certificates..."
+rm -rf $TMP_DIR
+mkdir -p $CERTS_DIR $TMP_DIR
+pushd $TMP_DIR 2>/dev/null 2>&1
+
+echo "[$(date)] $BIN/$$: configuring ca parameters"
+cat > ca.json <<EOF
+{
+  "signing": {
+    "default": {
+      "expiry": "8760h"
+    },
+    "profiles": {
+      "etcd": {
+        "usages": ["signing", "key encipherment", "server auth", "client auth"],
```

Comment:
We should be able to remove the "server auth" scope here

Reply:
Need to generate new cert for the client